app/paper/[slug]/page.tsx

```diff
@@ -52,12 +49,6 @@ export default async function Page({ params }: Props) {
 
       <hr className="my-12" />
 
-      <div className="mt-12 max-w-3xl space-y-3">
-        <img src={join(PATH_ASSETS, image)} alt="" />
-
-        <div className="font-sans">{publicationVenue}</div>
```

Comment:
Removed as this looked bad without the supporting image due to margin on the H2. 

Can we remove `publicationVenue` entirely from frontmatter? Not sure it's used anywhere else. @wackerow